src/components/Alert/Alert.mdx

```diff
@@ -19,6 +19,16 @@ Alerts allow you to display error messages relevant to an entire form or applica
   <Alert intent='error' message='Something went wrong.' />
 </Playground>
 
+### Different intent
+
+<Playground>
+  <Alert intent='success' message='This went great!' />
+  <br />
+  <Alert intent='warning' message='Are you sure?' />
```

Comment:
I would not pose a question in an alert, more something like:

"Editing this has also an effect on this"